.pending/improvements/sdk/4132-Fix-typos-and-s

```diff
@@ -0,0 +1 @@
+#4132 Fix typos and something outdated in CONTRIBUTING.md
```

Comment:
Imho, this doesn't warrant a pending log entry.

Reply:
Agree

Reply:
Done!